chart/kubeapps/Chart.yaml

```diff
@@ -25,4 +25,4 @@ maintainers:
 name: kubeapps
 sources:
   - https://github.com/kubeapps/kubeapps
-version: 7.1.3-dev1
+version: 7.1.4-dev0
```

Comment:
I expected that we'd do this manual change as part of the auto-generated PR triggered when the chart change lands at bitnami/charts and is synced back to our repo? But no issue, it's only dev related.

Reply:
Yes, generally we would change the chart version in the bitnami/chart repo, however, as we have perform some updates in the local chart (nginx 1.19 to 1.20, or re-generate the readme based on the lastest `redminator` tool version), I assumed we should also modify the local version (regardless of the change we also would need to to in the bitnami repo)